lib/temperature/index.js

```diff
@@ -93,6 +101,7 @@ TemperatureSensor.prototype.start = function start() {
   }
     
   try {
+    if (this.options.controller !== 'Simulator') {
```

Comment:
before we used

``sh
if (self.sensor.calibrate) {
__self...
``
then indent 2 spaces (not __) on next line and ongoing

Reply:
Comprend pas.